client/pages/checkout.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useRef, useEffect } from 'react';
+import { loadStripe } from '@stripe/stripe-js';
+import {
+  Elements
+} from '@stripe/react-stripe-js';
+import InjectedCheckoutForm from '../components/checkout-form';
+
+export default function CheckoutPage() {
+  const [clientSecret, setClientSecret] = useState('');
+  const stripePromiseRef = useRef(loadStripe('pk_test_51LunS5KbTmfJuQ2gX7JXiXBFsNz5hGUFYM2lwPy83t5vcp2klOQEdMs1icabJ1HGnYZ4Cxn0BgGeiiO2qmqdQwB700kLFc1AF4'));
```

Comment:
We should avoid hard coding api keys so that they can be easily swapped out. `process.env` does not exist in the browser, but Webpack can do a find-and-replace on the fly for us during bundling.

https://github.com/Learning-Fuze/final-project-instructions/blob/main/code-base.md#webpackconfigjs